test/e2e-v2/cases/mongodb/otel-collector-config.yaml

```diff
@@ -25,6 +25,23 @@ receivers:
              labels:
                cluster: replset
 
+##################################################################################################################
+# If you want to define instance name rather than using name in target, please reference to the following case.  #
+##################################################################################################################
+
+#        - job_name: 'mongodb-monitoring'
+#          metrics_path: "/metrics"
+#          scrape_interval: 10s
+#          static_configs:
+#            - targets: ['mongodb-exporter-1:9216']
+#              labels:
+#                cluster: replset
+#                service_instance_id: mongodb-1:27017
+#            - targets: ['mongodb-exporter-2:9216']
+#              labels:
+#                cluster: replset
+#                service_instance_id: mongodb-2:27017
```

Comment:
There is already a label `service_instance_id`, you add a same name label rather than use `relabel_configs`, is that works?

Reply:
Yes, it works.

![image](https://github.com/apache/skywalking/assets/16346644/920a5e88-dcfa-4fc2-b30b-013d84a92188)